### PR TITLE
LPS-158755 [Raylife AP] Creation Validations Form

### PR DESCRIPTION
**Pull request by edbbarros**

### CODE REVIEW

**Comment by edbbarros** [comment]
Sub-task: [LPS-158755](https://issues.liferay.com/browse/LPS-158755)

**Comment by edbbarros** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by edbbarros** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions. Please, change the commit name for the pattern we use.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 1]
```diff
@@ -13,23 +13,132 @@
...
+				{text ? text : 'This field is required'}
+			</div>
+		</div>
+	);
+
 	return (
 		<div className="container m-0 p-4">
 			<p className="font-weight-bolder mb-4 mt-0 text-paragraph-sm text-uppercase">
-				choose your coverages
+				CHOOSE YOUR COVERAGES
```
[the remark this answers is not in the document]
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:141]: Done!

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @edbbarros

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 2]
```diff
@@ -13,23 +13,132 @@
...
+			payload: isAbleToNextStep && hasAllRequiredFieldsToNextStep,
+			type: ACTIONS.SET_IS_ABLE_TO_NEXT,
+		});
+	};
+
+	useEffect(() => {
+		handleSaveChanges(form);
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [form]);
```
[the remark this answers is not in the document]
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:124]: it is needed to call the function "handleSaveChanges" with the parameter (form).

**Review by joanagrego — APPROVED** [review]

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by edbbarros** [comment]
ci:test:sf

**Comment by edbbarros** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]
LGTM :1st_place_medal: Thanks @edbbarros

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**